bin/git-unwip

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Based on scripts from git-utils:
+# * https://github.com/ddollar/git-utils/blob/master/git-unwip
+
+# Check if the last commit is a 'WIP' commit
+LAST_COMMIT=`git log -1 --pretty=%B | tr --delete '[:space:]'`
```

Comment:
```suggestion
LAST_COMMIT=`git log -1 --pretty=%B | tr -d '[:space:]'`
```

[BSD tr](https://man.freebsd.org/cgi/man.cgi?tr) doesn't seem to support `--delete`

Reply:
Well, that is unfortunate. I have been trying to use the full flag names in scripts, but you are right that BSD does not support it. :frowning_face: 

I will update to use `-d` as you suggested.